src/index.ts

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+import  KStream  from "./lib/dsl/KStream"
+import  KTable  from "./lib/dsl/KTable"
+import  KafkaFactory  from "./lib/KafkaFactory"
+import  KafkaStreams  from "./lib/KafkaStreams"
+import  KStorage  from "./lib/KStorage.js"
+import  KafkaClient  from "./lib/client/KafkaClient"
+
+module.exports = {
```

Comment:
I have fixed this up to use named exports as normal exports in https://github.com/nodefluent/kafka-streams/pull/166

Reply:
Oh.. I have missed that file.